.github/workflows/build.yml

```diff
@@ -284,6 +292,7 @@ jobs:
     name: Build test image
     needs:
       - diagnostics
+      - repo-metadata
       - prepare
```

Comment:
Should these be sorted alphabetically?

Reply:
When I put it in that order upstream it was based on order of appearance/dependence. The `prepare` job depends on the `repo-metadata` job so I put it in that order. I'm happy to adjust the order upstream to alphabetical if that's what we'd prefer.

Reply:
I think alphabetical order makes more sense, but I'm fine either way.